test/testcafe/spec/IdentityUnknownUser_spec.js

```diff
@@ -9,19 +10,36 @@ const mock = RequestMock()
   .onRequestTo('http://localhost:3000/idp/idx')
   .respond(unknownUser);
 
-fixture(`Unknown user form`)
-  .requestHooks(mock);
+  const registeredUserMock = RequestMock()
+  .onRequestTo('http://localhost:3000/idp/idx/introspect')
+  .respond(identify)
+  .onRequestTo('http://localhost:3000/idp/idx')
+  .respond(unknownUser)
+  .onRequestTo('http://localhost:3000/idp/idx/idx')
```

Comment:
this route seems incorrect `idp/idx/idx`.

Reply:
I think when you submit an existing user it would try to hit the same `http://localhost:3000/idp/idx` API again but you will respond with a different mock. You can avoid creating a new mock for registeredUser and just respond with [select-factor-authenticate.json](https://github.com/okta/okta-signin-widget/blob/master/playground/mocks/idp/idx/data/select-factor-authenticate.json) and just verify on that page that you no longer see the callout that you saw when you entered an invalid user.